backend/inspirehep/search/mappings/v7/records/hep.json

```diff
@@ -1497,6 +1497,11 @@
           "char_filter": ["dot_to_whitespace_normalizer"],
           "tokenizer": "standard",
           "type": "custom"
+        },
+        "page_artid_analyzer": {
```

Comment:
I don't think you're configuring the field to use this analyzer

Reply:
I just pushed it, you're too fast :P